File: examples/crypto/quote/go/getACryptocurrencyQuote.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/crypto"
)

// Get the last quote for a given crypto ticker
func GetAcryptocurrencyQuote() {
	cryptoService := crypto.NewCryptoService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := cryptoService.Quote(&crypto.QuoteRequest{
		Symbol: "BTCUSD",
	})
	fmt.Println(rsp, err)
}

File: examples/js/crypto/quote/getACryptocurrencyQuote.js
const { CryptoService } = require("m3o/crypto");

// Get the last quote for a given crypto ticker
async function getAcryptocurrencyQuote() {
  let cryptoService = new CryptoService(process.env.MICRO_API_TOKEN);
  let rsp = await cryptoService.quote({
    symbol: "BTCUSD",
  });
  console.log(rsp);
}

getAcryptocurrencyQuote();
